_posts/2021-12-08-cockpit-259.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Cockpit 259
+author: garrett
+date: '2021-12-08'
+tags: cockpit
+slug: cockpit-259
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 259, cockpit-machines 258, and cockpit-podman 38:
+
+## Storage: More information in table rows
+
+Table rows representing filesystems now show more information. You can
+now immediately see the mountpoint, the usage, and there are also
+links to the pages for volume groups.
+
+![screenshot of more information in table rows]({{ site.baseurl }}/images/259-storage-more-information-in-table-rows.png)
+
+## Cockpit Client
+
+Cockpit now ships [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient) via [Flatpak](https://flatpak.org/).
+
+If you are on a Linux desktop, this is now the most convenient and secure way (no open ports other than ssh) of accessing remote servers or your local machine with Cockpit.
```

Comment:
Does a screenshot make sense? We could take it from https://flathub.org/apps/details/org.cockpit_project.CockpitClient

Reply:
I don't know, it's just a screenshot of Cockpit...